kafka-inmemory/resources/kafka.yaml

```diff
@@ -23,6 +23,18 @@ spec:
           volumeMounts:
             - name: kafka-storage
               mountPath: /var/lib/kafka/
+          livenessProbe:
+            exec:
+              command:
+              - "/opt/kafka/kafka_healthcheck.sh"
+            initialDelaySeconds: 15
```

Comment:
How did you arrive at these delays and timeouts? Can we be sure they're always going to be the right value for all deployments? If not, perhaps we should parameterise them?

Reply:
There is no particular reason why 15 seconds delay and 5 second timeout are used. Too big values would slow down the reporting, too small could cause startup issues and trigger unnecessary restart. The behaviour might be different with busy and/or big clusters, so we might need to adjust it in the future. But we have to start somewhere and 15 and 5 seconds sounded good. We should observe how is behaves (both for Kafka and ZK) and fine tune it over the time. 

I added it as a parameter to the OpenShift templates.